public/components/map_container/map_container.tsx

```diff
@@ -241,8 +241,10 @@ export const MapContainer = ({
       (indexPattern) => indexPattern.id === selectedLayerConfig.source.indexPatternId
     );
     if (!findIndexPattern) {
-      // @ts-ignore
-      const newIndexPattern = await indexPatterns.get(selectedLayerConfig.source.indexPatternId);
+      const newIndexPattern = await services.data.indexPatterns.get(
+        // @ts-ignore
```

Comment:
Is this by mistake or intentional? having comment inside method parameter seems weird.

Reply:
Intentional for the below line